content/en/blog/_posts/2023-03-17-kubernetes-1.27-deprecations-and-removals.md

```diff
@@ -69,10 +69,9 @@ The following feature gates for [volume expansion](https://github.com/kubernetes
 
 ### Removal of `--master-service-namespace` command line argument
 
-The kube-apiserver accepts a deprecated command line argument,  `--master-service-namespace`, that specified where to create the Service named `kubernetes`
+The kube-apiserver accepts a deprecated command line argument, `--master-service-namespace`, that specified where to create the Service named `kubernetes`
 to represent the API server.
-Kubernetes v1.27 will remove that argument, which has been deprecated since the v1.6
-release.
+Kubernetes v1.27 will remove that argument, which has been deprecated since the v1.26 release.
```

Comment:
I think @harshitasao & others were partly right the first time. The informal deprecation was in 1.6 and 1.26 formally confirmed the deprecation.